scripts/install

```diff
@@ -217,6 +217,13 @@ ynh_print_info --message="Securing files and directories..."
 
 # Set permissions to app files
 chown -R $app: $final_path
+##add jail2ban file
+echo "[z-push] " > /etc/fail2ban/jail.d/z-push.conf
+echo "enable = true" >> /etc/fail2ban/jail.d/z-push.conf
+echo "port = http, https" >> /etc/fail2ban/jail.d/z-push.conf
+echo "filter = z-push " >> /etc/fail2ban/jail.d/z-push.conf
+echo "logpath = /var/log/z-push/z-push-error.log" >> /etc/fail2ban/jail.d/z-push.conf
+## that easy
```

Comment:
Unfortunately that's not the right way to proceed, because the current code is only there for install, corresponding file won't get removed during remove/uninstall, nor added during upgrade or restore ...

The proper way to do this is to use the yunohost helper `ynh_add_fail2ban_config` during install, upgrade, restore, and  `ynh_remove_fail2ban_config` during remove

https://yunohost.org/packaging_apps_helpers